controller.go

```diff
@@ -106,14 +119,16 @@ type driverData struct {
 type driverTable map[string]*driverData
 type networkTable map[types.UUID]*network
 type endpointTable map[types.UUID]*endpoint
-type sandboxTable map[string]*sandboxData
+type containerTable map[string]*sandboxData
```

Comment:
reviewed other parts of the PR and I dont see a need to have yet another type named containerTable. Its easy to understand and manage the objects if we consolidate them under a single collection. More comments down in the implementation.